chirp-impl/src/test/java/sample/chirper/chirp/impl/ChirpServiceTest.java

```diff
@@ -37,13 +50,13 @@ public static void tearDown() {
     }
 
     @Test
-    public void shouldPublishShirpsToSubscribers() throws Exception {
+    public void shouldPublishChirpsToSubscribers() throws Exception {
         ChirpService chirpService = server.client(ChirpService.class);
         LiveChirpsRequest request = new LiveChirpsRequest(TreePVector.<String>empty().plus("usr1").plus("usr2"));
-        Source<Chirp, ?> chirps1 = chirpService.getLiveChirps().invoke(request).toCompletableFuture().get(3, SECONDS);
+        Source<Chirp, ?> chirps1 = chirpService.getLiveChirps().invoke(request).toCompletableFuture().get(1, SECONDS);
         Probe<Chirp> probe1 = chirps1.runWith(TestSink.probe(server.system()), server.materializer());
         probe1.request(10);
-        Source<Chirp, ?> chirps2 = chirpService.getLiveChirps().invoke(request).toCompletableFuture().get(3, SECONDS);
+        Source<Chirp, ?> chirps2 = chirpService.getLiveChirps().invoke(request).toCompletableFuture().get(1, SECONDS);
```

Comment:
with the canary await in `setUp` these timeouts can be more aggressive.

Reply:
Don't you want to use `secondRequestOnEntityTimeout` here as well? And about on line 64?